src/main/java/com/github/princesslana/eriscasper/ErisCasper.java

```diff
@@ -36,20 +36,21 @@
   private final BotToken token;
 
   private final OkHttpClient httpClient = OkHttp.newHttpClient();
-  private final ObjectMapper jackson = Jackson.newObjectMapper();
-  private final Payloads payloads = new Payloads(jackson);
 
   private final Routes routes;
   private final Optional<ShardPayload> shard;
 
+  private final Gateway gateway;
+
   private ErisCasper(BotToken token, Optional<ShardPayload> shard) {
     this.token = token;
     this.shard = shard;
-    routes = new Routes(token, httpClient, jackson);
+    ObjectMapper jackson = Jackson.newObjectMapper();
```

Comment:
Just a note for later that we need to remove the need to do this. Everything should go through the Data.xxx methods.

Reply:
The only reason this is there is because it was there originally, I'm going to go into Routes and Payloads and replace the methods later, but for now, it can stay.